vnc_lite.html

```diff
@@ -179,16 +179,16 @@
             }
         }
 
-        var password = readQueryVariable('password', '');
-        var path = readQueryVariable('path', 'websockify');
+        const password = readQueryVariable('password', '');
+        const path = readQueryVariable('path', 'websockify');
 
         // Connect
         (function() {
```

Comment:
You should be able to nuke this thing as well as it was only there to create a scope AFAICT.